packages/analytics-js-integrations/src/integrations/DCMFloodlight/browser.js

```diff
@@ -99,6 +99,10 @@ class DCMFloodlight {
     return this.isLoaded();
   }
 
+  identify() {
+    logger.debug('[DCM Floodlight] identify:: method not supported');
```

Comment:
displayName will by default be in the log line
```suggestion
    logger.debug('identify:: method not supported');
```

Reply:
addressed